frontend/src/app/modules/workspace/personal/personal.component.scss

```diff
@@ -23,10 +23,6 @@ mat-list-item:hover {
   border: 2px solid #fb6d3a;
 }
 
-mat-list-item:nth-child(3) {
```

Comment:
Nice, but it creates this alignment issue
Can you try to deal with it?
![image](https://user-images.githubusercontent.com/20342130/154928497-32e35e46-7567-4b26-9d14-ffa38d24f7f3.png)

Reply:
oh, of course. i fix it